src/utils/sequelize.js

```diff
@@ -46,7 +46,7 @@ export const JsonUpdate = (self, col, callable, save, df = {}) => {
   }
 
   // eslint-disable-next-line no-param-reassign
-  self.dataValues[col] = JSON.stringify(value);
+  self.set(col, JSON.stringify(value));
```

Comment:
would love to know how to bypass getters and setters and just set the raw value of a column. The old way was just being completely ignored apparently.